scripts/easy-secrets-gen.sh

```diff
@@ -125,85 +139,31 @@ kubectl --namespace openstack \
     --type Opaque \
     --from-literal=username="ironic" \
     --from-literal=password="${IRONIC_RABBITMQ_PASSWORD}" \
-    --dry-run=client -o yaml > "${DEST_DIR}/secret-ironic-rabbitmq-password.yaml"
+    --dry-run=client -o yaml | secret-seal-stdin "${DEST_DIR}/secret-ironic-rabbitmq-password.yaml"
 
 [ ! -f "${DEST_DIR}/secret-ironic-db-password.yaml" ] && \
 kubectl --namespace openstack \
     create secret generic ironic-db-password \
     --type Opaque \
     --from-literal=password="${IRONIC_DB_PASSWORD}" \
-    --dry-run=client -o yaml > "${DEST_DIR}/secret-ironic-db-password.yaml"
+    --dry-run=client -o yaml | secret-seal-stdin "${DEST_DIR}/secret-ironic-db-password.yaml"
 
 [ ! -f "${DEST_DIR}/secret-ironic-keystone-password.yaml" ] && \
 kubectl --namespace openstack \
     create secret generic ironic-keystone-password \
     --type Opaque \
     --from-literal=username="ironic" \
     --from-literal=password="${IRONIC_KEYSTONE_PASSWORD}" \
-    --dry-run=client -o yaml > "${DEST_DIR}/secret-ironic-keystone-password.yaml"
+    --dry-run=client -o yaml | secret-seal-stdin "${DEST_DIR}/secret-ironic-keystone-password.yaml"
 
 if [ "x${DO_TMPL_VALUES}" = "xy" ]; then
     [ ! -f "${DEST_DIR}/secret-openstack.yaml" ] && \
     yq '(.. | select(tag == "!!str")) |= envsubst' \
         "./components/openstack-secrets.tpl.yaml" \
-        > "${DEST_DIR}/secret-openstack.yaml"
+        | secret-seal-stdin "${DEST_DIR}/secret-openstack.yaml"
```

Comment:
This file isn't a Kubernetes secret but is a Helm values.yaml with secrets. So Sealed Secrets won't work on it.

Reply:
Reverting; ArgoCD, multi-sourcing, and helm-secrets [aren't playing nice yet](https://github.com/jkroepke/helm-secrets/wiki/ArgoCD-Integration#multi-source-application-support). That's what we get for using bleeding edge technology.